core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/CarbonConstants.java

```diff
@@ -52,6 +52,8 @@ private CarbonConstants() {
     public static final String UI_PERMISSION_ACTION = "ui.execute";
     public static final String UI_USER_PERMISSIONS = "user-permissions";
 
+    public static final String IS_PASSWORD_TRIM_DISABLED = "DisablePasswordTrim";
```

Comment:
EnablePasswordTrim would be better IMO. Double negatives usually bring out a lot of headaches 😄

Reply:
fixed